eth/downloader/modes.go

```diff
@@ -19,7 +19,7 @@ package downloader
 import "fmt"
 
 // SyncMode represents the synchronisation mode of the downloader.
-type SyncMode int
+type SyncMode uint32
```

Comment:
maybe document that this is a uint32 since it's meant to be used with atomic operations